docs/docs/demo-dc-fabrics/readme.mdx

```diff
@@ -85,9 +85,10 @@ poetry run invoke load-schema load-data
 > [!NOTE]
 > The example below creates the Layer2 network service and a another Layer3 on topology fra05-pod1
 
+# TODO: Figure out the correct commands to execute these or does this require adding these in the UI instead?
```

Comment:
In the next release Infrahub will include a mutation to trigger the generators
the code is in the branch `release-1.2` already